pkgs/development/python-modules/memory_allocator/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, fetchPypi
+, buildPythonPackage
+, cython
+}:
+
+buildPythonPackage rec {
+  pname = "memory_allocator";
+  version = "0.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-UUcR71e3eAQIQpmWM+AVQxVtgHvrNjaIlHo5pURUln0=";
+  };
+
+  propagatedBuildInputs = [
+    cython
+  ];
+
```

Comment:
checkInputs and pythonImportsCheck missing.

Reply:
Thanks for the heads-up! I looked at this; this package has tests, and they are executed successfully when running `python setup.py test` (so the default `setuptoolsCheckPhase` runs the tests too). The tests are written in a way that  `pytest` (via `pytestCheckHook`) does not find anything to run. I'm open to suggestions, but I think leaving this implicit runs the most tests.